test/pom.xml

```diff
@@ -10,7 +10,6 @@
     <groupId>org.jboss.narayana.lra</groupId>
     <artifactId>lra-parent</artifactId>
     <version>1.0.0.Final-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
```

Comment:
I don't think you should remove this relativePath?

Reply:
../pom.xml is the default relativePath, so picked by default IIRC

Reply:
ah, OK - thank you for clarifying